common/buildings/hfe_buildings.txt

```diff
@@ -152,16 +167,79 @@ hfe_building_terraformer_hive_mind = {
 
 hfe_building_virtual = {
 	icon = building_virtual_entertainment_studios
+	base_cap_amount = 1
 	base_buildtime = @fetime
-	can_build = no
-	#can_demolish = no
+	can_demolish = no
+	can_be_ruined = no
+	can_be_disabled = yes
 	category = amenity
+	triggered_desc = {
+		text = VIRTUAL_FUTURE
+	}
+	potential = {
+		OR = {
+			owner = { has_country_flag = hfe_future }
```

Comment:
Allow me to explain why it works this way.

You see, the Virtual Life Complex is different from the Temple of Thoth. Instead of creating Jobs that go everywhere, it ties all unemployed Pops into a Job without caps. This is a mechanic that only functions with the building present. Thus, every time the Kad Republic colonizes a planet, it will obviously want to expand its Virtual Life Complex so that the new planet can enjoy its utopia.

ONLY the Kad Republic can build a Virtual Life Complex. Any nation can conquer and use it.

Reply:
This is single favor building intended only before awaking era.
It will be replaced by another building in future.

Reply:
I made major changes to that Building, which was unfinished when I started. I added flavor, balance, and purpose to it. These were all intentional acts.